qiita_pet/handlers/api_proxy/tests/test_artifact.py

```diff
@@ -310,20 +318,6 @@ def test_artifact_patch_request_errors(self):
                           'operations: replace'}
         self.assertEqual(obs, exp)
 
-    def test_artifact_delete_req(self):
```

Comment:
Why delete these tests? They're still important for making sure the call returns properly.